src/Domain/Infra/Doctrine/MappingConfigInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\Domain\Infra\Doctrine;
+
+/**
+ * @author Pascal Wacker <hello@pascalwacker.ch>
+ */
+interface MappingConfigInterface
```

Comment:
please remove this interface, we really dont need it. You can typehint for `MappingConfig` everywhere :)